pull/context.go

```diff
@@ -95,7 +95,10 @@ type Context interface {
 	Teams() (map[string]string, error)
 
 	// HasReviewers returns true if the Pull Request has reviewers
-	HasReveiwers() (bool, error)
+	HasReviewers() (bool, error)
+
+	// HasReviews returns true if the Pull Request has reviews
+	HasReviews() (bool, error)
```

Comment:
Does this need to be a separate function, or should it be combined with `HasReviewers`? It seems completely reasonable to me that `HasReviewers` would be true if there are requested reviewers or if there are actual reviews.

Reply:
Good point. I'll move this into `HasReviewers`.